Tests/PythonKitTests/PythonRuntimeTests.swift

```diff
@@ -194,10 +194,17 @@ class PythonRuntimeTests: XCTestCase {
                 XCTFail("non-Python error: \($0)")
                 return
             }
-            XCTAssertEqual(exception.__class__.__name__, "ZeroDivisionError")
+
+            // Something changed between Python 3.11 and 3.12 that causes the
+            // __class__ type to be different.
+            if (exception.__class__.__name__ == "str") {
```

Comment:
This seems like there is a deeper issue. Python 3.8, 3.9, 3.10 all report `ZeroDivisionError` when testing locally. This is the snippet that I used:

```python
import sys
print(sys.version)
try:
  1/0
except Exception as e:
  print(e.__class__.__name__)
```